modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld

```diff
@@ -3322,25 +3334,31 @@
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
 		<attribute>
-			<description></description>
+			<description>Deprecated as of 7.4.0, with no direct replacement.</description>
 			<name>componentId</name>
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
 		<attribute>
-			<description></description>
+			<description>Deprecated as of 7.4.0, with no direct replacement.</description>
 			<name>contributorKey</name>
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
 		<attribute>
 			<description></description>
+			<name>cssClass</name>
```

Comment:
Do we need to add `cssClass` and `labels`? Why would be want or need to extend API?

Reply:
Oh I see `cssClass` replaces `elementClasses`, that is fine :+1: 

My question then is just for `labels`.

Reply:
because that's something the component support and we weren't in previous tag.

https://clayui.com/docs/components/card.html#cardwithuser